core/src/types/github/graphql/enqueue_pull_request.graphql

```diff
@@ -3,3 +3,9 @@ mutation EnqueuePullRequest($id: ID!) {
         clientMutationId
     }
 }
+
+mutation ClosePullRequest($id: ID!) {

```

Comment:
Is this leftover from when you were helping me? :P

Reply:
oh yeah, lemme get rid of that